packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/http-server
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+import {RestServer, RestComponent} from '@loopback/rest';
+import {Application, ApplicationConfig} from '@loopback/core';
```

Comment:
http-server should not depend on `@loopback/core` and definitely not depend on `@loopback/rest`.

The integration tests should test `HttpServer` by calling its API, the tests are integration because under the hood, a real HTTP server will be created (integration with the system APIs is tested).